lib/src/icons.dart

```diff
@@ -12,470 +15,406 @@ class GYWIcon {
   /// The height (in pixels) of the icon
   final double height;
 
+  /// The path of the associated PNG file in the assets folder
+  final String pathPng;
+
+  /// The path of the associated SVG file in the assets folder
+  final String pathSvg;
+
   /// Creates an icon.
   const GYWIcon({
     required this.name,
     required this.filename,
-    required this.width,
-    required this.height,
+    this.width = 48,
+    this.height = 48,
+    this.pathPng = "assets/icons/unknown.png",
+    this.pathSvg = "assets/icons/unknown.svg",
   });
 
-  /// The path of the associated PNG file in the assets folder
-  String get pathPng => "assets/icons/$filename.png";
-
-  /// The path of the associated SVG file in the assets folder
-  String get pathSvg => "assets/icons/$filename.svg";
+  const GYWIcon._library({
+    required this.name,
+    required this.filename,
+    this.width = 48,
+    // ignore: unused_element
```

Comment:
If I don't put this, I have the following warning:

`info: A value for optional parameter 'height' isn't ever given. (unused_element at [flutter_gyw] lib/src/icons.dart:38)`